WooCommerce/Classes/POS/Presentation/Reusable Views/POSProgressViewStyle.swift

```diff
@@ -17,7 +17,8 @@ struct POSProgressViewStyle: ProgressViewStyle {
                 lineWidth: lineWidth,
                 lineCap: .butt,
                 circleColor: Color(.wooCommercePurple(.shade10)),
-                fillColor: Color(.wooCommercePurple(.shade50))
+                fillColor: Color(UIColor(light: .wooCommercePurple(.shade40),
+                                         dark: .wooCommercePurple(.shade30)))
```

Comment:
I updated this to match the updated app accent, which is `purple_40` in light theme and `purple_30` in dark theme.
This matches the color we use in the last Figma playground files (rNAclWFAdQRw7EXEcsNVai-fi-2533_5542) where we are are using `purple_40` on light theme.